sanitizer-macros/tests/struct.rs

```diff
@@ -38,7 +42,9 @@ fn sanitizer_check() {
         upper_case: String::from("hello, world"),
         camel_case: String::from("hello_world"),
         snake_case: String::from("helloWorld"),
-        screaming_snake_case: String::from("helloWorld"),
+        screaming_kebab_case: String::from("helloWorld"),
+        kebab_case: String::from("hello-World"),
+        screaming_snake_case: String::from("HELLO-WORLD"),
```

Comment:
These test cases aren't great - Example: "hello-World" -> "hello-world" can be addressed by lower_case as well. You should do "Hello, World" -> "hello-world" which considers capital letters & punctuation.

Reply:
Switch to `hello, world`